web-ng/README.md

```diff
@@ -3,40 +3,43 @@
 ## Dev Environment Setup
 
 Install ng CLI using npm:
+
 ```
 npm install -g @angular/cli
 ```
 
-Install angular-devkit:
-```
-npm install --save-dev @angular-devkit/build-angular
-```
+Install recent version of Node.js using nvm:
 
-Upgrade Node.js to the newest version using nvm: (Angular requires Node.js version 10.9.0 or later.)
 ```
-nvm install node --reinstall-packages-from=node
+nvm install 13.1.0 --reinstall-packages-from=node
```

Comment:
Why do we stick to a specific version instead of the newest one?

Reply:
A few reasons:
- So that our builds will be deterministic (especially CI builds and releases); at a particular commit the supported version is well-defined.
- For debugging developer issues. npm packages sometimes require different versions of node, and we'll get lots of questions from developers when they try to build with potentially newer versions than us.

Wdyt?